lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3InputStream.java

```diff
@@ -196,6 +196,10 @@ private void seekStream()
                 rangeRequest = request.toBuilder().range(range).build();
             }
             in = client.getObject(rangeRequest);
+            // a workaround for https://github.com/aws/aws-sdk-java-v2/issues/3538
```

Comment:
Is this a follow-up of https://github.com/trinodb/trino/pull/22469 ?

Reply:
Yes, see the PR description